recipes/ticcutils/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ticcutils" %}
+{% set version = "0.20" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/LanguageMachines/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "620f380829ae65b2418d0a2ae9f92d7f6b71eff5145b8cd7be7232889fc97bc5"
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - ticcutils
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - autoconf
+    - autoconf-archive
+    - automake
+    - libtool
+    - pkg-config
+  host:
+    - libtar
+    - libxml2
+    - icu
+    - boost
+    - zlib
+    - bzip2
+  run:
+    - libtar
+    - libxml2
+    - icu
+    - boost
+    - zlib
+    - bzip2
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/ticcutils  # [unix]
+    - test -f ${PREFIX}/lib/libticcutils.a  # [unix]
+
+about:
+  home: https://github.com/LanguageMachines/ticcutils
+  license: GPL-3.0
+  license_family: GPL
+  license_file: COPYING
+  summary: 'A generic utility library shared by Tilburg centre for Cognition and Communication (TiCC) software'
+  dev_url: https://github.com/LanguageMachines/ticcutils
+
+extra:
+  recipe-maintainers:
+    - egpbos
+    - jvdzwaan
+    - martinreynaert
```

Comment:
@jvdzwaan @martinreynaert are you OK with being maintainers

Reply:
I am OK with helping to OK releases. Do not expect me to actively help and develop this, please.

At this point in time I am not sure I really get what is being built here, or exactly how it is to further TICCLAT development, to be honest.

Reply:
Yes, OK with being a maintainer.